test/kubemark/start-kubemark.sh

```diff
@@ -394,6 +416,13 @@ current-context: kubemark-context")
     sed -i'' -e "s/{{kubemark_autoscaler_max_nodes}}/${KUBEMARK_AUTOSCALER_MAX_NODES}/g" "${RESOURCE_DIRECTORY}/addons/cluster-autoscaler.json"
   fi
 
+  # Kube DNS.
+  if [[ "${ENABLE_KUBEMARK_KUBE_DNS}" == "true" ]]; then
```

Comment:
Please make it "${ENABLE_KUBEMARK_KUBE_DNS:-}" so that bash doesn't exit with failure in case the variable isn't defined.

Reply:
done + same thing done for ENABLE_KUBEMARK_CLUSTER_AUTOSCALER